src/EnergyPlus/PlantPressureSystem.hh

```diff
@@ -105,7 +113,7 @@ struct PlantPressureSysData : BaseGlobalStruct
 
     bool InitPressureDropOneTimeInit = true;
     Array1D_bool LoopInit;
-    Array1D_bool FullParallelBranchSetFound = Array1D<bool>(2);
+    std::array<bool, static_cast<int>(DataPlant::LoopSideLocation::Num)> FullParallelBranchSetFound{false, false};
```

Comment:
Converted to std::array, so now it starts at 0, same as the LoopSideLocation std::array.